CMakeLists.txt

```diff
@@ -296,6 +303,15 @@ message(STATUS "MFEM build type: CMAKE_BUILD_TYPE = ${CMAKE_BUILD_TYPE}")
 message(STATUS "MFEM version: v${MFEM_VERSION_STRING}")
 message(STATUS "MFEM git string: ${MFEM_GIT_STRING}")
 
+if (SCOREC_FOUND)
+ message(STATUS "MFEM: using package SCOREC")
+ #define a header file with the MFEM_USE_SIMMETRIX preprocessor variable
+ set(MFEM_USE_SIMMETRIX ${SCOREC_gmi_sim_FOUND})
+ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/examples/pumi_config.h.in"
```

Comment:
Can we put `pumi_config.h.in` in the top-level `config` directory instead?

Reply:
Done.